spec/controller/api/v1/orders_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe  Api::V1::OrdersController, type: :request do
+  describe "GET #index" do
+    before do
+      @role = Role.create(id: 1, title: "User")
+      @user = User.create(id: 1, email: "abc@gmail.com", password: "123456789", role_id: @role.id)
+      @user_address = UserAddress.create(id: 1, address: "home", city: "Pune", pincode: 123456, state: "MH", country: "In", 
```

Comment:
if any object fails to create due to validation, we will use blank values. Ex, @role is failed to create @role.id will be nil. So nil data will be passed. It will take time to debug such issues.

If we use create!, it will raise errors in the first place. so it will help to fix the test immediately.